extensions/jdbc/jdbc-mariadb/runtime/src/main/java/io/quarkus/jdbc/mariadb/runtime/graal/SendPamAuthPacket_Substitutions.java

```diff
@@ -16,11 +14,6 @@
 @TargetClass(className = "org.mariadb.jdbc.plugin.authentication.standard.SendPamAuthPacket")
 public final class SendPamAuthPacket_Substitutions {
 
-    @Substitute
-    public void initialize(String authenticationData, byte[] seed, Configuration conf, HostAddress hostAddress) {
-        throw new UnsupportedOperationException("Authentication strategy 'dialog' is not supported in GraalVM");
```

Comment:
I think we probably need to throw the exception here instead:

https://github.com/mariadb-corporation/mariadb-connector-j/commit/734e7de199b6acad5ac0fdd994284d0f78526393#diff-b09663c0648950d8fca2cadd9fba74f071d49428b81cfe6877bbb8bd277ad3caR32